### DISCUSSION TITLE
Batch upload?

### DISCUSSION THREAD

**Discussion by mattjmeier** [abgulati/LARS#11] [General]
I was curious if there is a way (or a planned way) to upload, say, 1000 documents? Can I use the web upload button to add large numbers of files? Or is there a way via CLI, or interaction with the vector DB directly?

Sorry if this is a silly question - I managed to get it up and running quickly via docker bit haven't had a chance to play much yet. The answer to my question wasn't immediately clear to me so I thought it would be best to ask! 

Thanks!

**Comment by abgulati** [comment]
This feature is presently in-development so stay tuned for updates in the coming weeks!
